rust/azure_iot_operations_protocol/src/rpc_command/executor.rs

```diff
@@ -146,6 +153,25 @@ where
     }
 }
 
+/// Parse a [`azure_iot_operations_mqtt::aio::cloud_event::CloudEvent`] from a [`Request`].
+/// Note that this will return an error if the [`Request`] does not contain the required fields for a [`azure_iot_operations_mqtt::aio::cloud_event::CloudEvent`].
+///
+/// # Errors
+/// [`azure_iot_operations_mqtt::aio::cloud_event::CloudEventBuilderError::UninitializedField`] if the [`Request`] does not contain the required fields for a [`azure_iot_operations_mqtt::aio::cloud_event::CloudEvent`].
+///
+/// [`azure_iot_operations_mqtt::aio::cloud_event::CloudEventBuilderError::ValidationError`] if any of the field values are not valid for a [`azure_iot_operations_mqtt::aio::cloud_event::CloudEvent`].
+pub fn cloud_event_from_request<TReq: PayloadSerialize, TResp: PayloadSerialize>(
+    request: &Request<TReq, TResp>,
+) -> Result<
+    azure_iot_operations_mqtt::aio::cloud_event::CloudEvent,
```

Comment:
I don't love the overlapping naming between the protocol CloudEvent module and the mqtt aio CloudEvent module, any thoughts to help on that? I've been considering aliasing the module imports in these files to make things a little more brief?